src/services/print.js

```diff
@@ -550,6 +549,19 @@ ngeo.Print.prototype.getReportUrl = function(ref) {
 
 
 /**
+ * Get the capabilities of the report
```

Comment:
Can you please just rephrase this to "Get the print capabilities from MapFish Print."? Once done I'll merge your branch. Thanks a lot for your contribution.